src/GrammarApplicator_runRules.cpp

```diff
@@ -2725,6 +2730,7 @@ void GrammarApplicator::runGrammarOnWindow() {
 			Cohort* cc = current->cohorts[i];
 			cc->local_number = UI32(i); // Will also adjust newly restored cohorts
 			current->parent->cohort_map.insert(std::make_pair(cc->global_number, cc));
+			current->parent->dep_window.insert(std::make_pair(cc->global_number, cc));
```

Comment:
`dep_window` is for dependencies coming in from the stream - it is turned into `cohort_map` by `reflowDependencyWindow()` and then cleared. I question what led to inserting here - something didn't work without?

Reply:
That was from when I was trying to fix the interaction of `REMCOHORT` and `REMCOHORT IGNORE` and I didn't see that I had included it while I was committing this.